misc/toml-parser/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -0,0 +1,228 @@
+{
+    "root": "Node",
+    "type": "abstract",
+    "attributes": [
+        {
+            "name": "kind",
+            "type": "SyntaxKind"
+        },
+        {
+            "name": "width",
+            "type": "int"
+        }
+    ],
+    "nodes": [
+        {
+            "name": "ModulePartNode",
```

Comment:
Toml doesn't have a module concept. Shall we change `ModulePartNode ` to `DocumentNode`?